DogeNetWebClient/src/app/store/actions/conversation.actions.ts

```diff
@@ -0,0 +1,21 @@
+import { createAction, props } from '@ngrx/store';
+import { ConversationDetailsModel } from 'src/app/core/api/models';
+
+export const SET_CONVERSATIONS = '[Conversations] Set';
+export const ADD_CONVERSATION = '[Conversations] Add';
+export const DELETE_CONVERSATION = '[Conversations] Delete';
+export const CLEAR_CONVERSATIONS = '[Conversations] Clear';
+
+export const SetConversations = createAction(
+  SET_CONVERSATIONS,
```

Comment:
Do we need a separate variable for the action name?

Reply:
deleted